delfin/drivers/huawei/oceanstor/consts.py

```diff
@@ -119,6 +119,25 @@
         constants.DiskLogicalType.CACHE,
 }
 
+FS_WORM_COMPLIANCE = '1'
+FS_WORM_AUDIT_LOG = '2'
+FS_WORM_ENTERPRISE = '3'
+
+FS_HEALTH_NORMAL = '1'
+FS_TYPE_THICK = '0'
```

Comment:
FS_TYPE_THICK, FS_TYPE_THIN looks to be unused

Reply:
FS_TYPE_THIN is used in list_filesystems. (These are mapped oceanstor return codes)